### PR TITLE
Psuedo absolute changes

### PR DESCRIPTION
**Pull request by denniszollo**
Add a few new functions and change yaml definitions to support the psuedo absolutes in SBP.

### CODE REVIEW

File: sbp_generate/sbp.yaml [hunk 1]
```diff
@@ -139,8 +139,8 @@
                   desc: Fix mode
                   values:
                       - 0: Single Point Positioning (SPP)
-                      - 1: RTK
-
+                      - 1: RTK fixed 
+                      - 2: RTK float
```
Comment (fnoble) [inline sbp_generate/sbp.yaml]: Leaving a new line between each message definition makes for easier reading IMO.

File: sbp_generate/sbp.yaml [hunk 2]
```diff
@@ -189,8 +189,8 @@
                   desc: Fix mode
                   values:
                       - 0: Single Point Positioning (SPP)
-                      - 1: RTK
-
+                      - 1: RTK fixed
+                      - 2: RTK float
```
Comment (fnoble) [inline sbp_generate/sbp.yaml]: Likewise I'd suggest a new line here.

**Comment by fnoble** [comment]
Other than a couple pedantic style tweaks, looks great!

**Comment by denniszollo** [comment]
Changed sbp yaml and pdf (rebased the changes), ready for review and merge

**Pull request merged**